ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,68 @@
+<!-- 
+
+- Before submitting an issue please refer to https://lava-nc.org/developer_guide.html#how-to-contribute-to-lava
+
+- Please make sure you are posting an issue pertaining to the github.com/lava-nc/lava-dnf, for issues with lava please file in lava-nc/lava repository, for example github.com/lava-nc/lava/issues
+
+- Please do not submit support requests or "How to" questions here, use discussions Q&A https://github.com/lava-nc/lava-dnf/discussions/categories/q-a 
+
+- ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION. 
+
+-->
+
+<!-- Insert one sentence issue objective here, can be copied to PR. -->
+Objective of issue:
+
+<!-- Lava Version bug found in or Lava Version feature is targeting, remove unused versions from Lava Version below-->
+
+**Lava version:**
+- [ ] **0.3.0** (feature release)
+- [ ] **0.2.1** (bug fixes)
+- [x] **0.2.0** (current version)
+- [ ] **0.1.2**
```

Comment:
Shouldn't there be an additional question about the version of lava-dnf that this issue is targeting?

Reply:
Yes.